src/core/public/keyboard_shortcut/keyboard_shortcut_service.ts

```diff
@@ -54,7 +59,9 @@ export class KeyboardShortcutService {
       return;
     }
 
-    const key = this.keyParser.normalizeKeyString(shortcut.keys);
+    const key = isSequenceKeys(shortcut.keys)
+      ? this.sequenceHandler.normalizeKeyString(shortcut.keys)
+      : this.keyParser.normalizeKeyString(shortcut.keys);
 
     const namespacedId = this.getNamespacedId(shortcut);
```

Comment:
lets put 

`
const namespacedId = this.getNamespacedId(shortcut);
    if (this.namespacedIdToKeyLookup.has(namespacedId)) {
      throw new Error(
        `Shortcut "${shortcut.id}" from plugin "${shortcut.pluginId}" is already registered`
      );
    }
`
before we get the `key`